test/e2e/common/runtime.go

```diff
@@ -271,13 +272,13 @@ while true; do sleep 1; done
 					waiting:     true,
 				},
 				{
-					description: "should be able to pull image from gcr.io",
+					description: "should be able to pull image from gcr.io [LinuxOnly]",
```

Comment:
Since these don't seem inherently LinuxOnly, please add TODO comments to make it possible to also run them on Windows nodes

Reply:
Done, but this is the only test which is currently being Windows-ified (https://github.com/kubernetes/kubernetes/pull/72777), so I only added the TODO to this test.

All the other tests will require additional features or tests or both to be added in the future (e.g.: The non-root tests will be addressed when the RunAsUsername feature will be merged, but it will require new tests).